app/code/Magento/Catalog/Model/ResourceModel/AbstractResource.php

```diff
@@ -568,6 +568,6 @@ public function getAttributeRawValue($entityId, $attribute, $store)
             $attributesData = $_data[1];
         }
 
-        return $attributesData ? $attributesData : false;
+        return $attributesData!==false ? $attributesData : false;
```

Comment:
Please add spaces around binary operator.

Also, as to me this statement would be less confusing if you invert condition (`=== false` and swap ternary operator parts).